lib/ash_phoenix/gen/gen.ex

```diff
@@ -34,17 +34,18 @@ defmodule AshPhoenix.Gen do
 
     {parsed, _, _} =
       OptionParser.parse(rest,
-        strict: [resource_plural: :string, actor: :string, no_actor: :boolean]
+        strict: [
+          resource_plural: :string,
+          resource_plural_for_routes: :string,
+          actor: :string,
+          no_actor: :boolean
+        ]
       )
 
     domain = Module.concat([domain])
     resource = Module.concat([resource])
-
-    parsed =
-      Keyword.put_new_lazy(rest, :resource_plural, fn ->
-        plural_name!(resource, parsed)
-      end)
-
+    plural_name_fun = fn -> plural_name!(resource, parsed) end
+    parsed = Keyword.put_new_lazy(parsed, :resource_plural, plural_name_fun)
```

Comment:
This particular change doesn't seem necessary. Good to merge after we put it back.

Reply:
Right, but we can do this right?

```elixir
parsed = Keyword.put_new_lazy(parsed, :resource_plural, fn -> plural_name!(resource, parsed) end)
```

Reply:
Sure, it's the same. Moved the function to a variable so that it doesn't break line. My pet peeve. 👹 I'll update it that way then. 👌